pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -869,7 +868,13 @@ public void writeBrokerDataOnZooKeeper() {
             updateLocalBrokerData();
             if (needBrokerDataUpdate()) {
                 localData.setLastUpdate(System.currentTimeMillis());
-                zkClient.setData(brokerZnodePath, localData.getJsonBytes(), -1);
+
+                try {
+                    zkClient.setData(brokerZnodePath, localData.getJsonBytes(), -1);
+                } catch (KeeperException.NoNodeException e) {
+                    ZkUtils.createFullPathOptimistic(zkClient, brokerZnodePath, localData.getJsonBytes(),
```

Comment:
have you seen this behavior?
because `ModularLoadManagerImpl::start` tries to create `createZPathIfNotExists` in the beginning.

Reply:
Yes, Frequently appear in our recent performance tests, #5673 describe the problem